nomenclature/definitions/variable/technology/electrity-operation.yaml

```diff
@@ -0,0 +1,460 @@
+   unit: MWh
+   
+# Operation Cost per technologies
+# i.e.  operation cost of aggregated committments of plants, storages and demand-side systems
+
+Operation Cost|Electricity|Biomass:
+   description: Operation Cost of Biomass power plants
+   unit: €
+
+Operation Cost|Electricity|Biomass|Traditionnal:
+   description: Operation Cost of  Traditionnal Biomass power plants
+                biomass from local and pre-existing
+                resources mostly waste :domestic and agricultural waste, forest residues.
+   unit: €
+
+Operation Cost|Electricity|Biomass|New and imported:
+   description: Operation Cost of New+Imported biomass power plants
+                biomass from dedicated energy crops and biomass imports.
+   unit: €
+
+Operation Cost|Electricity|Nuclear:
+   description: Operation Cost of nuclear power plants
+   unit: €
+
+Operation Cost|Electricity|Geothermal:
+   description: Operation Cost of Geothermal power plants
+   unit: €
+   
+Operation Cost|Electricity|Gas:
+   description: Operation Cost of Gas power plants
+   unit: €
+   
+Operation Cost|Electricity|Gas|OCGT:
+   description: Operation Cost of Open Cycle Gas Turbine power plants
+   unit: €
+
+Operation Cost|Electricity|Gas|CCGT:
+   description: Operation Cost of Combined Cycle Gas Turbine power plants
+   unit: €
+   
+Operation Cost|Electricity|Coal:
+   description: Operation Cost of coal power plants
+   unit: €
+
+Operation Cost|Electricity|Lignite:
+   description: Operation Cost of Lignite power plants
+   unit: €
+
+Operation Cost|Electricity|Oil:
+   description: Operation Cost of Oil power plants
+   unit: €
+ 
+Operation Cost|Electricity|Hydro|Reservoir:
+   description: Operation Cost of "simple Hydro reservoir " units
+   unit: €
+   
+Operation Cost|Electricity|Hydro|Pumped Storage:
+   description: Operation Cost of "Pumped Storage " units
+   unit: €
+   
+Operation Cost|Electricity|Hydro|Run of River:
+   description: Operation Cost of "Run of River" units
+   unit: €
+
+Operation Cost|Electricity|Hydro|Ocean:
+   description: Operation Cost of Ocean Power plants
+   unit: €
+   
+Operation Cost|Electricity|Solar:
+   description: Operation Cost of Photovoltaic power plants
+   unit: €
+   
+Operation Cost|Electricity|Wind|OnShore:
+   description: Operation Cost of  Wind Onshore power plants
+   unit: €
+   
+Operation Cost|Electricity|Wind|OffShore:
+   description: Operation Cost of  Wind OffShore power plants
+   unit: €
+   
+Operation Cost|Electricity|Electricity Storage:
+   description: Operation Cost of energy storage systems, excl. hydro storage
+   unit: €
+
+# Automatic Frequency Restoration Reserve committed per technologies
+# i.e.  Automatic Frequency Restoration Reserve of aggregated committments of plants, storages and demand-side systems
+
+Automatic Frequency Restoration Reserve|Electricity|Biomass:
+   description: Automatic Frequency Restoration Reserve available for  Traditionnal Biomass power plants
```

Comment:
Is this the amount of AFRR available (could be used, i.e., maximum), or the amount actually contracted? And how will this relate to the amount of reserve required? That would be "Demand|..."?

Reply:
This is the amount committed, ie the maximum that can really be used if there is a problem on the grid. It is the result of the optimization (the optimisation decides from the Maximum Power of each plant which part to commit as active power, which part as aFRR, ....); It is different from the contracted amount, but the sum of all committed aFFR on a specific perimeter should be greater than the amount contracted.